dawn/prototype/mylib.hpp

```diff
@@ -303,118 +297,54 @@ class Grid {
 template <typename O, typename T>
 class Data {
 public:
-  explicit Data(size_t size) : data_(size) {}
-  T& operator()(O const& f) { return data_[f.id()]; }
-  T const& operator()(O const& f) const { return data_[f.id()]; }
+  explicit Data(size_t horizontal_size, size_t num_k_levels)
```

Comment:
```suggestion
  Data(size_t horizontal_size, size_t num_k_levels)
```
I would remove `explicit` here and below, when there is more than one constructor argument.